### ISSUE TITLE
Reward always 0

### ISSUE THREAD

**Issue by michael20at** [openai/retro#72]
## Issue summary

Reward (for x position) always 0

[Put a detailed description of the issue here.]

Hi, I modified the reward in scenario.json file for this game to variable "x", but no matter how far the character runs to the right, reward is always 0! Are there any changes in the dev build or am I missing something? I verified that I modified the right json file in the right folder, but still no reward!

```
import retro

env = retro.make('SonicTheHedgehog-Genesis', 'GreenHillZone.Act1')
env.reset()

while True:
    env.render()
    action = [0, 0, 1, 0, 0, 0, 0, 1, 1, 1, 0, 0]

    ob, rew, done, info = env.step(action)
    print("Action ", action, "Reward ", rew)
```

I'm running the 0.6 dev build wheel and just git cloned the retro folder (because install would through errors), games imported fine now, but might that be a cause?

## System information

- Win10
- Python 3.6
- Gym Retro 0.6 dev (windows wheel, no build)

**Comment by endrift** [comment]
Can I see the modified scenario.json? I wonder if you just have a syntax error or something.

**Comment by michael20at** [comment]
Sure, but I dont't think so:

```
{
  "done": {
    "variables": {
      "lives": {
        "op": "zero"
      }
    }
  },
  "reward": {
    "variables": {
      "x": {
        "reward": 1.0
      }
    }
  }
}
```

**Comment by michael20at** [comment]
Thanks, but the whole purpose of changing **scenario.json** was to get a different reward! Player should be rewarded just for going right, which it does in the simple agent, but reward is still zero no matter how far he goes.

There is a closed issue with a similar problem, where the proposed solution was to " import retro_contest.local, not retro", but there is no such modul in my build, and it is not mentioned anywhere else.

Any other ideas why reward doesn't work?

**Comment by endrift** [comment]
I'll take a closer look at this tomorrow.

**Comment by michael20at** [comment]
Never mind, figured it out myself, but thanks! I use WinPython, and installed the dev wheel as mentioned, then I git cloned the retro repository and changed the scenario reward. 

But that of course does not work because retro is using the scenario.json from the wheel and not the cloned version!, So after I rummaged through all the WinPython subfolders I found the right one in python-3.6.3.amd64\Lib\site-packages\retro\data\stable\, changed it and now it works!
